### ISSUE TITLE
Inconsistency between groups, channels & rooms

### ISSUE THREAD

**Issue by danger89** [RocketChat/Rocket.Chat#16982]
### Description:

Inconsistency of the naming of groups, channels, rooms within Rocket.chat app.

### Steps to reproduce:

1. Channel -> Info -> Says: "Room Info "
2. Admin panels -> "#rooms" 
3. Root url (you see channels & private groups)

### Expected behavior:

I except 1 name that is used at all locations including the admin panel. Instead of using all these different names.

### Actual behavior:

Instead of using channels and private groups, just call them either:

- Group
- Channel
- Rooms

And change all the GUI towards this name. So in this way of thinking instead of calling it `channel` and `private group`, eg. just call them `public channel` and `private channel`. And name the settings in the admin area `channels`. 

This would makes it so much consistent through out the application.

Ps. I do agree with the 'Direct Messages' type.

Let me know what you think! Maybe there was a good reason to do so? :confused:

**Comment by cyrillbolliger** [comment]
Good point. I totally agree, that a consistent naming would improve UX a lot. In the french translation we have the same issue as in english. In the german translation, the inconsistency between private and public channels is fixed, however discussions are sometimes called `Diskussionen` and sometimes `Unterhaltung`. I think `Diskussion` would be the appropriate word to use in all places.

**Comment by politbuero-kampagnen** [comment]
we have been rolling out rocketchat to a large user base and this incoherent naming created much confusion among even experienced users.

**Comment by politbuero-kampagnen** [comment]
We are still observing that this inconsistency (that could be easily changed in Labels/Doc) is heavily disturbing new (and even seasoned) users, thereby creating unnecessary support calls or even make people quit the service at all :-(
